ChairManager/ConfigManager.cpp

```diff
@@ -178,10 +177,198 @@ void ConfigManager::drawXMLConfigNode(pugi::xml_node node) {
 }
 
 void ConfigManager::setDirty(const std::string& modName, bool dirty) {
+//    std::scoped_lock lock(m_modConfigsMutex);
```

Comment:
Why is the mutex commented out?

Reply:
Commented out bc of deadlock when setDirty is called in the draw function. Not ideal but it's part of a recursive call chain which is hard to mutex.